src/blockChainManager/encodeSHA256.ts

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable import/prefer-default-export */
+import { SHA256 } from 'crypto-js';
+
```

Comment:
all theses functions should be in a single file -> src/service/blockChainManager.ts @EliTheSurfer